test/functionalTest/cases/0876_entity_dates/entity_dates_overridden_by_user_subs.test

```diff
@@ -33,27 +33,27 @@ accumulatorStart --pretty-print
 #
 # (Test focuses on dateModified, but dateCreated is expected to work in a similar way)
 #
-# E1 notifications in steps 1 to 9 should not happen (they has been marked with [*]), as the
```

Comment:
Diff is pretty noisy in this file. I'd suggest to have a look to the final file and ensure it correct regarding what is told in section "User attributes or metadata matching builtin name" of ngsiv2_implementation_notes.md file.

Reply:
NTC